dev/acf/group_5e3b16e3f0830.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Resolving since ACF field conditions don't support conditions based on page template, and it's probably not worth the time/overhead to make this work.